app/invocation/workflow_commands.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { build_event_stream } from "../../proto/build_event_stream_ts_proto";
+import InvocationModel from "./invocation_model";
+import WorkflowCommandsCard from "./workflow_commands_card";
+
+export type WorkflowCommandsProps = {
+  model: InvocationModel;
+};
+
+export type BazelCommandResult = {
+  invocation: build_event_stream.WorkflowConfigured.IInvocationMetadata;
+  durationMillis?: number;
+};
+
+export default class WorkflowCommands extends React.Component<WorkflowCommandsProps> {
+  render() {
+    const configuredEvent = this.props.model.workflowConfigured!;
+    const completedEventsById = this.props.model.workflowCommandCompletedByInvocationId;
+
+    const completed: BazelCommandResult[] = [];
+    const failed: BazelCommandResult[] = [];
+    const notRun: BazelCommandResult[] = [];
+
+    let prevTimeMillis = Number(this.props.model.started.startTimeMillis);
+    for (const invocation of configuredEvent.invocation) {
+      const completedEvent = completedEventsById.get(invocation.invocationId);
+      if (!completedEvent) {
```

Comment:
modifying the invocation is kind of gnarly - can we either not pass invocations ids for these or pass a param to WorkflowCommandsCard on whether or not the links should be clickable?

Reply:
Passed a `linksDisabled` prop to WorkflowCommandsCard